.github/workflows/pr-main-warning.yml

```diff
@@ -1,12 +1,12 @@
 name: PR Main Branch Warning
 on:
-  pull_request:
+  pull_request_target:
     types: [opened]
     branches:
       - main
 
 permissions:
-  issues: write
```

Comment:
when I tried this, I needed the 'issues' permission because the 'issues' GH APIs are being used?